test/helpers/mentor_dashboard_helper_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+class MentorDashboardHelperTest < ActionView::TestCase
+  test "time_ago_in_days calculates time in days" do
+    assert_equal "2 days", time_ago_in_days(Time.now - 2.days)
+  end
+
+  test "time_ago_in_days calculates time less than one day in hours or minutes" do
+    assert_equal "about 2 hours", time_ago_in_days(Time.now - 2.hours)
```

Comment:
I'm still not quite clear on why the method is called `time_ago_in_days` instead of `time_ago_in_words`. How would you feel about overriding Rails implementation of `time_ago_in_words`?

Reply:
I had tried earlier to override the method in an intializer file but this time I just did it in a regular helper file and it worked fine. I guess there is a risk in overriding a method globally but in this case I think the new implementation is an improvement so that would make it worthwhile.

Reply:
If we're overriding the method, we want to override it everywhere, not just in a subset of the project, as consistency is important.